.github/workflows/node-test.yml

```diff
@@ -0,0 +1,21 @@
+name: 'DHIS2'
+
+on: [push]
+
+jobs:
+    pr:
+        name: Unit Tests
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v1
+            - uses: actions/setup-node@v1
+              with:
+                  node-version: 12.x
+
+            - name: Install
+              run: yarn install
+
+            - name: Test
+              run: yarn test
```

Comment:
Might be a good idea to add a `yarn build` step here, so that we make sure the build doesn't break in PRs?